tests/requirements.txt

```diff
@@ -1,5 +1,3 @@
 # These are Python requirements needed to run the functional tests
-# 1.6.1 fails with 'testinfra is in an unsupported or invalid wheel'
-# see https://github.com/philpep/testinfra/issues/201
-testinfra==1.6.0
+testinfra==1.7.1
```

Comment:
Does testinfra 1.7 not work with the latest Ansible 2.4? The commit message makes no mention of this

Reply:
Correct, it does not work with 1.6, https://github.com/philpep/testinfra/issues/249.
Will edit the commit. Thanks Ken.